### ISSUE TITLE
Log the date in the status bar when creating custom date

### ISSUE THREAD

**Issue by math2001** [FichteFoll/InsertDate#36]
Hello!

There's everything in the title pretty much...

Thanks for your plugin anyway!

Matt

**Comment by FichteFoll** [comment]
Good idea. I considered doing a live preview, but just showing it in the status bar is much easier and should work just fine. 

Just to know I understand this correctly, you mean  when entering a format in the input panel, right?

**Comment by math2001** [comment]
> you mean when entering a format in the input panel, right?

yes, this:

![image](https://cloud.githubusercontent.com/assets/15224242/21764092/67532344-d6b5-11e6-93ee-77da38afb522.png)
